pkg/storage/cacher_test.go

```diff
@@ -335,7 +338,6 @@ func TestFiltering(t *testing.T) {
 	}
 	defer watcher.Stop()
 
-	verifyWatchEvent(t, watcher, watch.Added, podFoo)
```

Comment:
Same here - can you please pass fooCreated.ResourceVersion -1 instead in line 335?